back/engines/commercial/id_clave_unica/app/lib/id_clave_unica/clave_unica_omniauth.rb

```diff
@@ -15,6 +15,13 @@ def profile_to_user_attrs(auth)
       if (ln = auth.dig('extra', 'raw_info', 'name', 'apellidos'))
         info[:last_name] = ln.join(' ')
       end
+
+      # We save formatted RUT in IdCard to be able to use this table for IdCard verification
```

Comment:
So, in `IdCard`, we hash RUT in the `44.444.444-4` format. And in `Verification::Verification`, we hash RUT in the `44444444` format (it's problematic to change it because we already have many verified users). 

I hope this little advantage described in the comment is worth it.